data/lock_block/tags/blocks/full_block_lock.json

```diff
@@ -3,15 +3,13 @@
         "minecraft:composter",
         "minecraft:crafting_table",
         "minecraft:cartography_table",
-        "minecraft:fletching_table",
```

Comment:
Ideally, I'd like to keep this, in case then next big Minecarft version(s) bring functionality to the Fletching table finally.